Exercise_2.py

```diff
@@ -6,10 +6,44 @@ def __init__(self, data):
  
 class Stack:
     def __init__(self):
+        self.head=None
+               
+
         
     def push(self, data):
```

Comment:
As you r making stack using LinkedList, so all operations should be in done in constants space right, so add/remove node at head of list